static/templates/admin_user_list.handlebars

```diff
@@ -6,45 +6,47 @@
   <td>
     <span class="email">{{email}}</span>
   </td>
-  {{#if is_bot}}
+  {{#if is_user_admin}}
```

Comment:
I think we should rename it to something like `current_user_is_admin` or something more clear.